crmsh/utils.py

```diff
@@ -2287,4 +2253,104 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+class InterfaceInfo(object):
```

Comment:
I would change to `InterfacesInfo` as you store there all of them

Reply:
Changed